src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -262,10 +268,14 @@ private String getPaymentReferenceFromTransaction(String uri, String passthrough
         }
     }
 
-    private void setDescriptionFields(FilingApi filing) {
-        String formattedRegistrationDate = dateNowSupplier.get().format(formatter);
+    private void setDescriptionFields(FilingApi filing, boolean isUpdateFiling) {
+        String formattedDate = dateNowSupplier.get().format(formatter);
         filing.setDescriptionIdentifier(filingDescriptionIdentifier);
-        filing.setDescription(filingDescription.replace("{registration date}", formattedRegistrationDate));
+        if (isUpdateFiling) {
+            filing.setDescription(updateFilingDescription.replace("{date}", formattedDate));
+        } else {
+            filing.setDescription(filingDescription.replace("{date}", formattedDate));
```

Comment:
for the registration path, I think it should still be replacing {registration date} instead of {date}

Reply:
I have updated the env variables of both the OVERSEAS_ENTITIES_FILING_DESCRIPTION and OVERSEAS_ENTITIES_UPDATE_FILING_DESCRIPTION to be {date} at the end to simplify

Reply:
ok thanks, make sure to get them updated in the release notes when deploying